packages/gestalt/src/TextField.js

```diff
@@ -48,6 +48,10 @@ type Props = {|
    * A unique identifier for the input.
    */
   id: string,
+  /**
+   * Optionally specify the type of data that might be entered by the user while editing the element or its contents. This allows a browser to display an appropriate virtual keyboard. See the [enterKeyHint variant](https://gestalt.pinterest.systems/web/textfield#EnterKeyHint) for more info.
```

Comment:
Probably more like "This allows _mobile browsers_ to display an appropriate…", yes? Virtual keyboards aren't ever a thing on desktop?

Reply:
DONE